benchmark/locust/run_locust.py

```
@@ -60,19 +63,33 @@ def __init__(self, config: LocustConfig):
-        """Check if the NeMo Guardrails server is up before running tests."""
-        url = f"{self.config.host}/health"
+    def _get(self, url: str) -> httpx.Response:
+        """Issue a preflight GET, translating transport failures into RuntimeError."""
         log.debug("Checking service is up at %s", url)
 
         try:
             # Try a simple request to verify the server is accessible
-            response = httpx.get(url, timeout=5)
+            return httpx.get(url, timeout=5)
         except httpx.ConnectError as e:
             raise RuntimeError(f"ConnectError accessing {url}: {e}")
```

Ruff (0.16.1)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)